app/etl/ga/views_processor.rb

```diff
@@ -38,7 +38,7 @@ def load_metrics
     date_to_s = date.strftime("%F")
 
     conn.execute(load_metrics_query(date_to_s))
-    conn.execute(clean_up_query)
+    clean_up_query
```

Comment:
What about renaming this method to: `clean_up_events!`

Reply:
Good idea, thanks @pmanrubia. Done!